week_1/week_1-ritapragya-biswas

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        .top-align {
+    height: 125px;
+    vertical-align: top;
+}
+
+
+body {
+    background-color: rgb(198, 255, 255);
+}
+
+h2 {
+    color: rgb(255, 145, 71);
+}
+
+h4 {
+    color: brown;
+}
+
+textarea {
+    background-color: rgb(255, 255, 205);
+}
+
+textarea:hover {
+    background-color: rgb(161, 255, 210);
+}
+
+input[type=text] {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    background-color: rgb(255, 255, 205);
+}
+
+input[type=text]:hover {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    background-color: rgb(161, 255, 210);
+}
+
+input[type=checkbox]:hover {
+    box-shadow: 0px 0px 10px #1300ff;
+}
+    </style>
+</head>
+
+<body>
+    <h2>My Guest Book </h2>
+    Please sign my guest book. Thanks!
+    <table>
+        <tr>
+            <td>
+                <form>
+                    <label for="name">
+                        <h4>What is your name?</h4>
+                    </label>
+            </td>
+            <td>
+                <input type="text" id="name" name="name"><br>
+                </form>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <form>
+                    <label for="email">
+                        <h4>What is your email?</h4>
+                    </label>
+            </td>
+            <td>
+                <input type="text" id="email" name="email"><br>
+                </form>
+            </td>
+        </tr>
+        <tr>
+            <td class="top-align">
+                <form>
+                    <label for="feedback">
+                        <h4>Please check all that apply:</h4>
+                    </label>
+            </td>
+            <td>
+                <input type="checkbox" id="feedback1" name="feedback1">
+                <label for="feedback1"> I really like your website</label><br>
+                <input type="checkbox" id="feedback2" name="feedback2">
+                <label for="feedback2"> One of the best sites I've seen</label><br>
+                <input type="checkbox" id="feedback3" name="feedback3">
+                <label for="feedback3"> I sure wish my site looked as good as yours</label><br>
+                <input type="checkbox" id="feedback4" name="feedback4">
+                <label for="feedback4"> I have no taste and I'm pretty dense so I don't like your site</label><br>
+                </form>
+            </td>
+        </tr>
```

Comment:
Hey @ritapragya-biswas good job.
Some corrections -
- A single [form tag](https://www.w3schools.com/html/html_forms.asp) should be used for all related input field.
- Make sure HTML files have .html extinction (eg index.html)

Reply:
resolved the extension issue and will keep the first point in mind in future. thank you